src/Search.js

```diff
@@ -21,7 +21,20 @@ export default class Search extends Component {
                     <option value='desc'>High to low</option>
 
                 </select>
-
+                {
+                    <button
+                        disabled={this.props.pageNumber === 1}
+                        onClick={this.props.handleDecrement}>
+                        Prev
+            </button>
+                }
+                {
+                    <button
+                        onClick={this.props.handleIncrement}
+                        disabled={this.props.pageNumber === Math.ceil(this.props.count / 20)}>
```

Comment:
you could make the 20 a variable (added in your state and fetch.get()) if you want to...